tools/dhtnode.cpp

```diff
@@ -96,7 +96,7 @@ void cmd_loop(std::shared_ptr<DhtRunner>& dht, std::map<std::string, dht::indexa
             break;
         } else if (op == "h" || op == "help") {
             print_help();
-            continue;
+            continue;;
```

Comment:
yup that's compile, that's interpreted as a : 
continue;
;
But there is still an error